log.go

```diff
@@ -80,6 +100,122 @@ type eventLogger struct {
 	// TODO add log-level
 }
 
+type activeSpanKeyType struct{}
+
+var activeSpanKey = activeSpanKeyType{}
+
+type activeSpan struct {
+	isFinished bool
+}
+
+func (el *eventLogger) Start(ctx context.Context, name string) context.Context {
+	el.Event(ctx, name, LoggableMap{"system": el.system})
```

Comment:
Isn't this redundant? I thought we were now writing open tracing information to our main log now.

Reply:
yup, sorry missed this..